### PR TITLE
Create Herndon City Outline + Recommendations

### PR DESCRIPTION
**Pull request by RameshKatkam**
My First Commit. :)

This Pull Request Closes issue #45 
### Adding Restaurants and activities.
- Does anyone have a favorite type of food? 
- Outdoor or indoor activities? 

@githubteacher

### CODE REVIEW

File: githubteacher-HerndonCity.va [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Herndon City :heart:
+
+## Restaurents
+
```
Comment (RameshKatkam) [inline githubteacher-HerndonCity.va]: I Love to Eat Pizza. :)

File: githubteacher-HerndonCity.va [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## Herndon City :heart:
```
Comment (doppeldimmykumont) [inline githubteacher-HerndonCity.va:1]: Where is herndon city?

File: githubteacher-HerndonCity.va [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+## Restaurents
+
+## Things to do:
```
Comment (pritish-devurkar) [inline githubteacher-HerndonCity.va:7]: Whats #1 on this list?
 @RameshKatkam

**Comment by jdavis1998** [comment]
What is your favorite restaurant?

File: githubteacher-HerndonCity.va (hunk 2 above)
Comment (RameshKatkam) [inline githubteacher-HerndonCity.va:1]: @doppelDimmykumont  North Virginia closer To Washington D.C Area

File: githubteacher-HerndonCity.va (hunk 3 above)
Comment (RameshKatkam) [inline githubteacher-HerndonCity.va:7]: North Virginia closer To Washington D.C Area @findpritz

**Pull request merged**